sips/sip-030/sip-030-wallet-interface.md

```diff
@@ -0,0 +1,527 @@
+# Preamble
+
+SIP Number: `030`
+
+Title: Integration of a Modern Stacks Wallet Interface Standard
+
+Authors: [aryzing](https://github.com/aryzing), [janniks](https://github.com/janniks), [kyranjamie](https://github.com/kyranjamie),  [m-aboelenein](https://github.com/m-aboelenein)
+
```

Comment:
Missing 'Consideration' field in the Preamble:  Please add this field and specify the appropriate consideration(s), such as 'Technical'.

Reply:
For the `Type` field is there a category "non consensus breaking standard"?